include/lightweight_filtering/OutlierDetection.hpp

```diff
@@ -44,7 +44,7 @@ class OutlierDetectionBase{
     outlier_ = d_ > mahalanobisTh_;
     if(outlier_){
       outlierCount_++;
-      VLOG(2) << "Detected outlier - mahalanobis (d / threshold):"  << d_
+      VLOG(5) << "Detected outlier - mahalanobis (d / threshold):"  << d_
```

Comment:
Hope this is fine

Reply:
should we keep it in vlog 1? it might be still pretty relevant when debugging